File: Search/maven/src/test/java/Search/FileReaderTest.java
package Search;

import static org.junit.Assert.assertEquals;

import java.io.IOException;

import org.junit.Test;

public class FileReaderTest {
    private String testPath = "maven\\src\\test\\resources\\TestDocs";

    @Test
    public void hasNextTest() throws IOException {
            FileReader reader = new FileReader(testPath);
            assertEquals(reader.hasNext(), true);
    }

    @Test
    public void NextTest() throws IOException {
        
            FileReader reader = new FileReader(testPath);
            String actual = reader.next();
            String expected = "The sky , at sunset , looked like a carnivorous flower .";
            assertEquals(expected, actual);
    
    }
}

File: Phase05/Search/Search.Test/UtilsTest/FileReaderTest.cs
using Xunit;
using Search.Utils;

namespace Search.Test.UtilsTest
{
    public class FileReaderTest
    {
        private const string TestPath =  "resources";

        [Fact]
        public void MoveNextTest()
        {
            
            FileReader reader = new FileReader(TestPath);
            Assert.Equal(reader.MoveNext(),true);
        }

        [Fact]
        public void CurrentTest()
        {
            var reader = new FileReader(TestPath);
            reader.MoveNext();

            string actual = reader.Current();
            string expected = "The sky , at sunset , looked like a carnivorous flower .";
            
            Assert.Equal(actual,expected);
        }
        
    }
}